File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/Paginacion.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for Paginacion complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="Paginacion">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="totalRegistros" type="{http://www.w3.org/2001/XMLSchema}int" minOccurs="0"/>
 *         &lt;element name="paginaActual" type="{http://www.w3.org/2001/XMLSchema}int" minOccurs="0"/>
 *         &lt;element name="registrosPorPagina" type="{http://www.w3.org/2001/XMLSchema}int" minOccurs="0"/>
 *         &lt;element name="mostrandoDesde" type="{http://www.w3.org/2001/XMLSchema}int" minOccurs="0"/>
 *         &lt;element name="mostrandoHasta" type="{http://www.w3.org/2001/XMLSchema}int" minOccurs="0"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "Paginacion", propOrder = {
    "totalRegistros",
    "paginaActual",
    "registrosPorPagina",
    "mostrandoDesde",
    "mostrandoHasta"
})
public class Paginacion {

    protected Integer totalRegistros;
    protected Integer paginaActual;
    protected Integer registrosPorPagina;
    protected Integer mostrandoDesde;
    protected Integer mostrandoHasta;

    /**
     * Gets the value of the totalRegistros property.
     * 
     * @return
     *     possible object is
     *     {@link Integer }
     *     
     */
    public Integer getTotalRegistros() {
        return totalRegistros;
    }

    /**
     * Sets the value of the totalRegistros property.
     * 
     * @param value
     *     allowed object is
     *     {@link Integer }
     *     
     */
    public void setTotalRegistros(Integer value) {
        this.totalRegistros = value;
    }

    /**
     * Gets the value of the paginaActual property.
     * 
     * @return
     *     possible object is
     *     {@link Integer }
     *     
     */
    public Integer getPaginaActual() {
        return paginaActual;
    }

    /**
     * Sets the value of the paginaActual property.
     * 
     * @param value
     *     allowed object is
     *     {@link Integer }
     *     
     */
    public void setPaginaActual(Integer value) {
        this.paginaActual = value;
    }

    /**
     * Gets the value of the registrosPorPagina property.
     * 
     * @return
     *     possible object is
     *     {@link Integer }
     *     
     */
    public Integer getRegistrosPorPagina() {
        return registrosPorPagina;
    }

    /**
     * Sets the value of the registrosPorPagina property.
     * 
     * @param value
     *     allowed object is
     *     {@link Integer }
     *     
     */
    public void setRegistrosPorPagina(Integer value) {
        this.registrosPorPagina = value;
    }

    /**
     * Gets the value of the mostrandoDesde property.
     * 
     * @return
     *     possible object is
     *     {@link Integer }
     *     
     */
    public Integer getMostrandoDesde() {
        return mostrandoDesde;
    }

    /**
     * Sets the value of the mostrandoDesde property.
     * 
     * @param value
     *     allowed object is
     *     {@link Integer }
     *     
     */
    public void setMostrandoDesde(Integer value) {
        this.mostrandoDesde = value;
    }

    /**
     * Gets the value of the mostrandoHasta property.
     * 
     * @return
     *     possible object is
     *     {@link Integer }
     *     
     */
    public Integer getMostrandoHasta() {
        return mostrandoHasta;
    }

    /**
     * Sets the value of the mostrandoHasta property.
     * 
     * @param value
     *     allowed object is
     *     {@link Integer }
     *     
     */
    public void setMostrandoHasta(Integer value) {
        this.mostrandoHasta = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/Paginacion.php
<?php

namespace Semantikos\ClientBundle\API;

class Paginacion
{

    /**
     * @var int $totalRegistros
     */
    protected $totalRegistros = null;

    /**
     * @var int $paginaActual
     */
    protected $paginaActual = null;

    /**
     * @var int $registrosPorPagina
     */
    protected $registrosPorPagina = null;

    /**
     * @var int $mostrandoDesde
     */
    protected $mostrandoDesde = null;

    /**
     * @var int $mostrandoHasta
     */
    protected $mostrandoHasta = null;

    
    public function __construct()
    {
    
    }

    /**
     * @return int
     */
    public function getTotalRegistros()
    {
      return $this->totalRegistros;
    }

    /**
     * @param int $totalRegistros
     * @return Paginacion
     */
    public function setTotalRegistros($totalRegistros)
    {
      $this->totalRegistros = $totalRegistros;
      return $this;
    }

    /**
     * @return int
     */
    public function getPaginaActual()
    {
      return $this->paginaActual;
    }

    /**
     * @param int $paginaActual
     * @return Paginacion
     */
    public function setPaginaActual($paginaActual)
    {
      $this->paginaActual = $paginaActual;
      return $this;
    }

    /**
     * @return int
     */
    public function getRegistrosPorPagina()
    {
      return $this->registrosPorPagina;
    }

    /**
     * @param int $registrosPorPagina
     * @return Paginacion
     */
    public function setRegistrosPorPagina($registrosPorPagina)
    {
      $this->registrosPorPagina = $registrosPorPagina;
      return $this;
    }

    /**
     * @return int
     */
    public function getMostrandoDesde()
    {
      return $this->mostrandoDesde;
    }

    /**
     * @param int $mostrandoDesde
     * @return Paginacion
     */
    public function setMostrandoDesde($mostrandoDesde)
    {
      $this->mostrandoDesde = $mostrandoDesde;
      return $this;
    }

    /**
     * @return int
     */
    public function getMostrandoHasta()
    {
      return $this->mostrandoHasta;
    }

    /**
     * @param int $mostrandoHasta
     * @return Paginacion
     */
    public function setMostrandoHasta($mostrandoHasta)
    {
      $this->mostrandoHasta = $mostrandoHasta;
      return $this;
    }

}
